codex/blockexchange/protobuf/message.nim

```diff
@@ -5,6 +5,15 @@
 
 import pkg/libp2p/protobuf/minprotobuf
 
+const
+  DefaultMaxSize = 100 * 1 shl 20 
```

Comment:
Could we change this to something easier to read at a glance? 

You could do `10.MiBs.int` [units.nim](https://github.com/codex-storage/nim-codex/blob/a89938478a1a0c58b48ff8ec3bddfd5c96b51522/codex/units.nim#L49C6-L49C10).